systemtest/ingest_test.go

```diff
@@ -123,10 +123,11 @@ func TestIngestPipelineEventDuration(t *testing.T) {
 	}
 
 	tests := []test{{
-		// No transaction.* field, no update.
-		source: `{"@timestamp": "2022-02-15", "observer": {"version": "8.2.0"}, "processor": {"event": "transaction"}}`,
+		// No transaction.* field, no event.duration: set transaction.duration.us to zero.
+		source:                        `{"@timestamp": "2022-02-15", "observer": {"version": "8.2.0"}, "processor": {"event": "transaction"}}`,
+		expectedTransactionDurationUS: 0.0,
```

Comment:
Note for reviewers: there's a small functional change here, which won't actually ever manifest in reality - transactions and spans always have a `transaction` or `span` object defined.